frontend/components/ViewAllHostsLink/_styles.scss

```diff
@@ -0,0 +1,13 @@
+.view-all-hosts-link {
+  display: flex;
+  align-items: center;
+  padding: $pad-small $pad-xxsmall; // larger clickable area
+
+  &__icon {
```

Comment:
Ive been thinking about this css problem more. The problem is

`.icon` and `.view-all-hosts-link__icon` have the same specificity and rely on the ordering of the stylesheets for the correct styling to be applied. I don't think we should rely on this ordering and when we override the generic component we will need to use this type of selector to make sure there is enough specificity.

```scss
.view-all-hosts-link {
    .icon {
        ....
    }
}
```

tbh, I'm not a huge fan of doing this. Any idea how we can make sure these styles get applied here?

Reply:
Yaas, good call. I toyed with it and since we're now using display: flex, I was able to style properly using an inline-flex and a gap. Check it out.